src/Bicep.LangServer/Snippets/Templates/param-defaults-allowed-values.bicep

```diff
@@ -2,4 +2,4 @@
 @allowed([
   /*$3*/3
 ])
-param /*${1:Identifier}*/Identifier /*${2:Type}*/int = /*$4*/4
\ No newline at end of file
+param /*${1:Identifier}*/Identifier /*${2:Type}*/int = /*$3*/3
```

Comment:
Note that the `$integer` &* `${Integer:...}` placeholders are creating tab stops for snippets. I don't think it makes sense to re-use them (in fact, I'm surprised the snippet tests didn't catch this!)

If you're just looking to fix this test, you should be able to update this value and re-generate baselines: https://github.com/Azure/bicep/blob/d9380c420b73ae01571177210f894c11e5b7baa2/src/Bicep.LangServer.IntegrationTests/Files/SnippetTemplates/param-defaults-allowed-values/main.bicep#L4